packages/tracing/src/browser/metrics.ts

```diff
@@ -236,11 +236,13 @@ export class MetricsInstrumentation {
         return;
       }
 
-      const timeOrigin = msToSec(performance.timeOrigin);
       const startTime = msToSec(entry.startTime as number);
       logger.log('[Measurements] Adding LCP');
       this._measurements['lcp'] = { value: metric.value };
-      this._measurements['mark.lcp'] = { value: timeOrigin + startTime };
+      if (browserPerformanceTimeOrigin !== undefined) {
```

Comment:
`browserPerformanceTimeOrigin` should always exist when we can actually report LCP and FID.

I'd just replace `performance.timeOrigin` above with `browserPerformanceTimeOrigin`.

I see we use `performance.timeOrigin` directly in ember... maybe we change that too, cc @k-fish.

https://github.com/getsentry/sentry-javascript/blob/3eb9e059d08339e542985e8a97eb563ad0812e28/packages/ember/addon/instance-initializers/sentry-performance.ts#L302

https://github.com/getsentry/sentry-javascript/blob/3eb9e059d08339e542985e8a97eb563ad0812e28/packages/ember/addon/instance-initializers/sentry-performance.ts#L312

Reply:
@rhcarvalho  typescript told me that `browserPerformanceTimeOrigin` could be `undefined`, but I'll coerce it as a `number`

Reply:
Yeah typescript... it is only undefined if `performance` is also undefined.